packages/gamut-templates/src/Alert/types.ts

```diff
@@ -0,0 +1,14 @@
+export type BannerCTA = {
+  href?: string;
+  text: string;
+  onClick: () => void;
+  disabled?: boolean;
+};
+
+export enum BannerTypes {
```

Comment:
The normal naming convention would be singular, since the value is only ever one of it.

```suggestion
export enum BannerType {
```